api-gateway/src/main/resources/application.yml

```diff
@@ -15,6 +15,9 @@ gateway:
     url: http://demo:5401
   demo-cpg:
     url: http://demo-cpg:5500
+  neo4j-browser:
+    url: http://save-cloud:7474
```

Comment:
why save-cloud? does neo4j have such name?

Reply:
Weirdly enough, it does. From `neo4j-svc.yaml`:
```yaml
metadata:
  name: "{{ $.Release.Name }}"
```

Reply:
can we override it? or will leave as is?

Reply:
how we can override their names? I think we need to report it